12 add subtract matrices.cpp

```diff
@@ -30,7 +33,7 @@ main()
 	if((row_1==row_2)&&(col_1==col_2)){
 		printf("\nThe matrices are compatible for addition and subtraction");
 		printf("\nTo add: Press 1\nTo subtract: Press 2\n");
-		int a;																			//To know what we want to do with the matrices
+		int a;												//To know what we want to do with the matrices

```

Comment:
Comment is too far in the right. Most common practice is to place it on separate line.
Do this for other comments like this as well
```suggestion
 		// To know what we want to do with the matrices
		int a;
```